code/modules/client/preference_setup/general/05_background.dm

```diff
@@ -72,28 +68,32 @@
 		"records_security" = pref.sec_record,
 		"char_id" = pref.current_character,
 		"nt_relation" = pref.nanotrasen_relation,
-		"home_system" = pref.home_system,
 		"citizenship" = pref.citizenship,
 		"religion" = pref.religion,
 		"id" = pref.current_character,
 		"ckey" = PREF_CLIENT_CKEY
 	)
 
 /datum/category_item/player_setup_item/general/background/sanitize_character()
-	if(!pref.home_system)
-		pref.home_system	= "Unset"
 	if(!pref.citizenship)
-		pref.citizenship	= "None"
+		pref.citizenship	= CITIZENSHIP_BIESEL
 	if(!pref.religion)
-		pref.religion		= "None"
+		pref.religion		= RELIGION_NONE
+
+	var/datum/species/S = all_species[pref.species]

```

Comment:
I am now wondering if this is safe. HMMMM.

Specifically, which will get sanitized first.

Reply:
The first one is incase there is no citizenship or religion, the second one is if somehow the religion or citizenship is not possible for this species.